DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditRegistrationPrompt.cshtml

```diff
@@ -0,0 +1,110 @@
+  </div>
+</div>
+
+<form class="nhsuk-u-margin-bottom-8" method="post" novalidate asp-action="EditRegistrationPrompt">
+  <div class="hidden-submit">
+    <button name="action" class="nhsuk-button" value="save">Save</button>
+  </div>
+
+  <input type="hidden" asp-for="OptionsString"/>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l">
+      <div class="nhsuk-u-font-weight-bold">
+        Prompt:
+      </div>
+    </div>
+    <div class="nhsuk-grid-column-one-quarter nhsuk-heading-l nhsuk-u-font-weight-normal">
+      <input type="hidden" asp-for="Prompt"/>
+      @Model.Prompt
+    </div>
+  </div>
+
+  @if (errorHasOccurred) {
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <vc:error-summary order-of-property-names="@new []{ nameof(EditRegistrationPromptViewModel.OptionsString), nameof(EditRegistrationPromptViewModel.Answer)}"/>
+      </div>
+    </div>
+  }
+
+  <div class="nhsuk-grid-row nhsuk-u-padding-bottom-4">
+    <div class="nhsuk-grid-column-one-half">
+      <div class="nhsuk-checkboxes">
+        <div class="nhsuk-checkboxes__item">
+          <input class="nhsuk-checkboxes__input" id="Mandatory" type="checkbox" asp-for="Mandatory">
+          <label class="nhsuk-label nhsuk-checkboxes__label" for="Mandatory">
+            This prompt is mandatory
+          </label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  @if (!string.IsNullOrEmpty(Model.OptionsString)) {
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <table class="nhsuk-table">
+          <caption class="nhsuk-table__caption">Configured answers:</caption>
+          <thead role="rowgroup" class="nhsuk-table__head">
+          <tr role="row">
+            <th role="columnheader" class="" scope="col">
+              Answer
+            </th>
+            <th role="columnheader" class="" scope="col">
+              Action
+            </th>
+          </tr>
+          </thead>
+          <tbody class="nhsuk-table__body">
+          @foreach (var answer in Model.Options) {
+            <tr role="row" class="nhsuk-table__row">
+              <td class="nhsuk-table__cell">@answer</td>
+              <td class="nhsuk-table__cell ">
+                <button name="action" class="nhsuk-button nhsuk-button--secondary nhsuk-u-padding-bottom-1 nhsuk-u-padding-top-1 nhsuk-u-margin-bottom-0" value="@("delete" + Model.Options.IndexOf(answer))">Remove</button>
+              </td>
+            </tr>
+          }
+          </tbody>
+        </table>
+      </div>
+    </div>
+  } else {
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <partial name="NoConfiguredAnswers"/>
+      </div>
+    </div>
+  }
+
+  <div class="nhsuk-grid-row divider">
+    <div class="nhsuk-grid-column-one-half">
+      <vc:text-input asp-for="Answer"
+                     label="Add another answer?"

```

Comment:
Should this be changed to something like "Add a new answer"? Just because "Add _another_ answer" doesn't sound right to me for the case where no answers exist.

Reply:
Yeah I think that is better